clients/src/main/java/org/apache/kafka/common/network/SSLTransportLayer.java

```diff
@@ -667,13 +676,21 @@ private int readFromAppBuffer(ByteBuffer dst) {
         return remaining;
     }
 
-    private int packetBufferSize() {
+    protected int netReadBufferSize() {
```

Comment:
The test is using different values for `netReadBufferSize` and `netWriteBufferSize` in some cases. Whether that is useful or not, I'll let @rajinisivaram explain.